autogen/agentchat/contrib/capabilities/vision_capability.py

```diff
@@ -3,16 +3,16 @@
 
 from autogen.agentchat.assistant_agent import ConversableAgent
 from autogen.agentchat.contrib.capabilities.agent_capability import AgentCapability
-from autogen.agentchat.contrib.img_utils import (
+from autogen.agentchat.contrib.multimodal_conversable_agent import MultimodalConversableAgent
+from autogen.agentchat.conversable_agent import colored
```

Comment:
is this `colored` function the same as the one in `formatting_utils`?

Reply:
Yes. Good catch. Removed the unnecessary imports.